CHANGELOG_INSTALLER_LINUX.md

```diff
@@ -1,5 +1,8 @@
 # Changelog for Linux-Installer-Script
 
+## 2020-06-19 
```

Comment:
Also here it is not the installer changelog  :-) The change you did needs to be pushed as new npm package version

Reply:
We should probably merge the changelogs and just write whether the fixer, installer, base package or both were updated.

Reply:
I would agree to merge the changelogs, as at the moment, some changes like this (or them before from me) wouldn't be noted, but are actually changed.

Reply:
But how to version the basepackage then? Or have changelog like "Installer 2020-06-19 / Package 1.2.3" or how?

Reply:
How about this?

## v1.2.3
Change that affects the package itself

## 2020-06-23
### Fixer
Did stuff

### Installer
Did some other stuff